tds/src/main/webapp/WEB-INF/templates/commonFragments.html

```diff
@@ -37,7 +37,7 @@
     </div>
 
     <div th:fragment="footer-content">
-      <h4><a class="static" th:href="${installUrl}" th:text="${installName}"></a> at <a class="static" th:href="${hostInstUrl}" th:text="${hostInst}"></a> see <a class="static" th:href="${contextPath} + '/serverInfo.html'"> Info </a></h4>
```

Comment:
We want to keep the info link, as it links to a page with general and contact info for the individual server, as specified in threddsConfig.xml (for example: http://thredds.ucar.edu/thredds/serverInfo.html).

That said, in 5.0, the `ServerInfoController` was changed such that the url should reference the /info service, so the correct url should actually be something like

https://thredds-dev.unidata.ucar.edu/thredds/info/serverInfo.html

Would you mind adding that fix to this PR?